src/invocation/agent.rs

```diff
@@ -290,6 +297,9 @@ pub enum ReceiveError<
 > where
     <S as Store<T, DID, V, C>>::InvocationStoreError: fmt::Debug,
 {
+    #[error("missing delegation: {0}")]
+    MissingDelegation(Cid),
```

Comment:
This is slightly ambiguous: is there a missing delegation in the `prf` field vs from the store. Have you considered something like `CannotFindDelegation(Cid)`?

Reply:
Yeah I like your suggestion!

Reply:
I took the freedom to adjust it to say `DelegationNotFound` :v: